modules/management-console-agent/src/main/java/org/apache/ignite/agent/Agent.java

```diff
@@ -299,6 +299,14 @@ private void connect0() {
     private void connect() {
         if (!cfg.isEnabled()) {
             log.info("Skip start Management Console agent on coordinator, because it was disabled in configuration");
+            log.info("You can use control script to enable Management Console agent");
+
+            return;
+        }
+
+        if (F.isEmpty(cfg.getConsoleUris())) {
+            log.info("Skip start Management Console agent on coordinator, because the server URI is not set");
+            log.info("You can use control script to setup server URI");
```

Comment:
```suggestion
            log.info("You can use `control.sh|bat` to setup server URI");
```

Reply:
I can't find usage of such term in code, but in same time if you enter 'ignite control script' in google  you can find link to documentation https://apacheignite-tools.readme.io/docs/control-script